go.mod

```diff
@@ -6,9 +6,9 @@ toolchain go1.22.0
 
 require (
 	github.com/gin-gonic/gin v1.9.1
-	github.com/konveyor/analyzer-lsp v0.7.0-alpha.2.0.20250414173818-69f16ed24d65
-	github.com/konveyor/tackle2-addon v0.7.0-alpha.2.0.20250416134249-7c0f33d51619
-	github.com/konveyor/tackle2-hub v0.7.0-alpha.2.0.20250417030359-32aba8021fd6
+	github.com/konveyor/analyzer-lsp v0.7.0-alpha.2.0.20250528164516-ec5a199590bf
+	github.com/konveyor/tackle2-addon v0.7.0-rc.1.0.20250604115244-17140aad3dd4
+	github.com/konveyor/tackle2-hub v0.7.0-alpha.2.0.20250603155042-56f863e431f6
```

Comment:
THIS ^ NEEDS TO BE UPDATED AFTER https://github.com/konveyor/tackle2-hub/pull/832 HAS MERGED.